doc/source/theory/unres_cfd-dem/unres_cfd-dem.rst

```diff
@@ -0,0 +1,150 @@
+
+In unresolved CFD-DEM drag is calculated using correlations (frequently called drag models). The drag models implemented in Lethe are described in the `unresolved CFD-DEM parameters guide <https://lethe-cfd.github.io/lethe/parameters/unresolved_cfd-dem/cfd_dem.html>`_.
+
+Volume Average Navier-Stokes
+-----------------------------
+
+Since we represent the fluid at a meso-scale, the quantities calculated for the subdomains are averages among its volume. Additionally, as the volume of fluid is a fraction of the subdomain, the porosity (or void fraction) is taken into account. To do this, we apply the Volume Average Navier-Stokes (VANS) equations to represent the fluid phase. Mainly, the VANS equations are presented in two different formulations, so called Model A (or Set II) and Model B (or Set I).
+
+For both models, considering incompressible fluid, the continuity is:
+
+.. math::
+    \frac{\partial \varepsilon_f}{\partial t} + \nabla \cdot \left ( \varepsilon_f \mathbf{u} \right ) = 0
+
+where:
+
+* :math:`\mathbf{u}` is the the fluid velocity vector;
+* :math:`\varepsilon_f` is the void fraction.
+
+Models A and B differ from each other in the way the momentum equation is calculated. In Model A, we consider that pressure is in both phases, while for Model B the pressure is only in the fluid:
+
+Model A:
+
+.. math:: 
+    \rho_f \left ( \frac{\partial \left ( \varepsilon_f \mathbf{u} \right )}{\partial t} + \nabla \cdot \left ( \varepsilon_f \mathbf{u} \otimes \mathbf{u} \right ) \right ) = -\varepsilon \nabla p + \varepsilon \nabla \cdot \tau + \mathbf{F}_{pf}^A
+
+Model B:
+
+.. math:: 
+    \rho_f \left ( \frac{\partial \left ( \varepsilon_f \mathbf{u} \right )}{\partial t} + \nabla \cdot \left ( \varepsilon_f \mathbf{u} \otimes \mathbf{u} \right ) \right ) = -\nabla p + \nabla \cdot \tau + \mathbf{F}_{pf}^B
+
+where:
+
+* :math:`\rho_f` is the density of the fluid;
+* :math:`p` is the pressure;
+* :math:`\tau` is the shear stress;
+* :math:`\mathbf{F}_{pf}^A` and :math:`\mathbf{F}_{pf}^B` are the source terms representing the forces applied back in the fluid due to the interaction with particles for Models A and B, respectively.
+
+For Model A, since the pressure term corresponds to a 'fluid fraction of the pressure', we can write the interaction term as:
+
+.. math:: 
+    \mathbf{F}_{pf}^A = -\frac{1}{V_{\Omega}}\sum_{i}^{n_p}\left ( \mathbf{f}_{pf, i} - \mathbf{f}_{\nabla p, i} - \mathbf{f}_{\nabla \cdot \tau, i} \right )
+
+while for Model B, since the pressure is totally in the fluid, we write:
+
+.. math:: 
+    \mathbf{F}_{pf}^B = -\frac{1}{V_{\Omega}}\sum_{i}^{n_p}\left ( \mathbf{f}_{pf, i} \right )
+
+where :math:`n_p` is the number of particles inside the subdomain :math:`\Omega` with volume :math:`V_{\Omega}`.
```

Comment:
```suggestion
where :math:`n_p` is the number of particles inside the subdomain :math:`\Omega` with volume :math:`V_{\Omega}`.
```
I guess here subdomain means cell? If that's what you mean, please define it properly before (like significantly above) and explain that in what follows a subdomain is generally a cell.